llm/codellama/endpoint.yaml

```diff
@@ -27,9 +27,7 @@ setup: |
     conda activate codellama
   fi
 
-  # We have to manually install Torch otherwise apex & xformers won't build
   pip list | grep torch || pip install "torch>=2.0.0"
-
```

Comment:
Just checking - is this intended as a part of this PR?

Reply:
Yes, the comment is staled. Also, just checked that vllm has already pinned the torch version to be `2.1.2`, so we don't need to manually install torch here. Removing the torch installation as well.